src/opnsense/mvc/app/controllers/OPNsense/Firewall/Api/FilterController.php

```diff
@@ -458,4 +458,17 @@ public function getInterfaceListAction()
 
         return $result;
     }
+
+    public function flushInspectCacheAction()
+    {
+        if (!$this->request->isPost()) {
+            return ['status' => 'error', 'message' => gettext('Invalid request method')];
+        }
+
+        // The "!" prelude flushes any cached script_output before execution
```

Comment:
I don't think we need to explain it like this, especially since the action is called "flush" :)

```suggestion
```